### PR TITLE
Use OSS chart as a subchart

### PR DESCRIPTION
**Pull request by jedcunningham**
The Astronomer chart will use the open source chart as a subchart, then add back some platform specific resources as necessary.

### CODE REVIEW

File: requirements.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 dependencies:
-  - name: postgresql
-    version: 6.3.12
-    repository: "@stable"
-    condition: postgresql.enabled
+  - name: airflow
+    version: 1.1.0-rc1
+    repository: https://github.com/astronomer/airflow/releases/download/helm-chart%2Fv1.0.1-dev1/
```
Comment (kaxil) [inline requirements.yaml:4]:
```suggestion
    repository: https://github.com/astronomer/airflow/releases/download/oss-helm-chart%2F1.1.0-rc1/
```

**Comment by kaxil** [comment]
Some unit tests are failing

**Comment by kaxil** [comment]
@jedcunningham / @aliotta Can one of you look into failing uni tests please

File: README.md [hunk 2]
```diff
@@ -172,113 +175,27 @@ workers:
...
+| `workers.autoscaling.targetCPUUtilization`            | Target CPU Utilization of workers                                                                         | `80`                                                              |    |
+| `workers.autoscaling.targetMemoryUtilization`         | Target Memory Utilization of workers                                                                      | `80`                                                              |    |
 | `extraObjects`                                        | Extra K8s Objects to deploy (these are passed through `tpl`). More about [Extra Objects](#extra-objects). | `[]`                                                              |    |
-| `webserver.extraContainers`                           | Add additional containers to webserver pod(s)                                                             | `[]`                                                              |    |
+| `sccEnabled`                                          | Enable security context constraints required for OpenShift                                                | `false`                                                           |    |
```
Comment (jedcunningham) [inline README.md:199]: `authSidecar` stuff is missing here.

File: UPDATING.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    {{- with .Values.airflow.labels }}
+    {{ toYaml . | indent 4 }}
+    {{- end }}
+    spec:
+      accessModes:
+        - ReadWriteMany
+      resources:
+        requests:
+          storage: 5Gi
+```
```
Comment (jedcunningham) [inline UPDATING.md:123]: Need a section for authSidecar, no?

File: requirements.yaml [hunk 4]
```diff
@@ -1,5 +1,4 @@
 dependencies:
-  - name: postgresql
-    version: 6.3.12
-    repository: "@stable"
-    condition: postgresql.enabled
+  - name: airflow
+    version: 1.1.0-rc1
+    repository: https://github.com/astronomer/airflow/releases/download/oss-helm-chart%2F1.1.0-rc1/
```
Comment (jedcunningham) [inline requirements.yaml:4]:
```suggestion
    version: 1.1.0
    repository: https://airflow.apache.org/
```

File: templates/_helpers.yaml [hunk 5]
```diff
@@ -3,6 +3,9 @@
   # Hard Coded Airflow Envs
   - name: ASTRONOMER__AIRFLOW__WORKER_LOG_RETENTION_DAYS
     value: "3"
```
Comment (jedcunningham) [inline templates/_helpers.yaml:5]: What in Airflow uses this env var? Is this in the right place?

File: templates/airflow-scc-anyuid.yaml [hunk 6]
```diff
@@ -21,6 +21,7 @@ users:
 - system:serviceaccount:{{ .Release.Namespace }}:{{ .Release.Name }}-scheduler-serviceaccount
 - system:serviceaccount:{{ .Release.Namespace }}:{{ .Release.Name }}-webserver-serviceaccount
 - system:serviceaccount:{{ .Release.Namespace }}:{{ .Release.Name }}-worker-serviceaccount
+# TODO: ^^ service account names are different now, _and_ can be changed
```
Comment (jedcunningham) [inline templates/airflow-scc-anyuid.yaml:24]:
```suggestion
- system:serviceaccount:{{ .Release.Namespace }}:{{ include "scheduler.serviceAccountName" . }}
- system:serviceaccount:{{ .Release.Namespace }}:{{ include "webserver.serviceAccountName" . }}
- system:serviceaccount:{{ .Release.Namespace }}:{{ include "worker.serviceAccountName" . }}
```

Should the worker serviceaccount be conditional based on executor? We don't even create it unless we are using CeleryExecutor or KubernetesExecutor.

**Review by jedcunningham — COMMENTED** [review]
Also surprised to see `templates/scheduler-deployment.yaml` isn't deleted?

**Comment by aliotta** [comment]
> Also surprised to see `templates/scheduler-deployment.yaml` isn't deleted?

My bad that was lost in the rebase. Re-removing makes the unit tests pass again.

**Review by kaxil — APPROVED** [review]

**Pull request merged**